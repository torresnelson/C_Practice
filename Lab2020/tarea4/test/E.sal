1>Encolado.
2>Encolado.
3>Encolado.
4>Encolado.
5>Vacio.
6>
7>Union.
8>Intersección.
9>Establecida la asociación.
10>Establecida la asociación.
11>p NO está llena.
12>p está vacia.
13>2
14>Desencolado.
15>cavl NO está vacia.
16>13
17>Altura de avl: 3.
18>13
 |  - 14
19>No pertenece.
20>Pertenece.
21>3 6 
22>No existe asociacion.
23>15.00
24>No lleno.
25>Vacia.
26>No está.
27>10.
28>Fin.
==7088== 8 bytes in 1 blocks are definitely lost in loss record 1 of 3
==7088==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==7088==    by 0x110591: crearConjunto() (conjunto.cpp:12)
==7088==    by 0x10DB77: interseccionDeConjuntos(repConjunto*, repConjunto*) (usoTads.cpp:11)
==7088==    by 0x10BD27: main (principal.cpp:765)
==7088== 
==7088== 40 (8 direct, 32 indirect) bytes in 1 blocks are definitely lost in loss record 3 of 3
==7088==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==7088==    by 0x110591: crearConjunto() (conjunto.cpp:12)
==7088==    by 0x11068B: unionDeConjuntos(repConjunto*, repConjunto*) (conjunto.cpp:32)
==7088==    by 0x10DCCE: interseccionDeConjuntos(repConjunto*, repConjunto*) (usoTads.cpp:24)
==7088==    by 0x10BD27: main (principal.cpp:765)
==7088== 
