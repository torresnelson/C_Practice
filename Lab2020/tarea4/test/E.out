1>Encolado.
2>Encolado.
3>Encolado.
4>Encolado.
5>Vacio.
6>
7>Union.
8>Intersección.
9>Establecida la asociación.
10>Establecida la asociación.
11>p NO está llena.
12>p está vacia.
13>2
14>Desencolado.
15>cavl NO está vacia.
16>13
17>Altura de avl: 3.
18>13
 |  - 14
19>No pertenece.
20>Pertenece.
21>3 6 
22>No existe asociacion.
23>15.00
24>No lleno.
25>Vacia.
26>No está.
27>10.
28>Fin.
